dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -48,6 +49,20 @@
 	#address-cells = <1>;
 	#size-cells = <1>;
 
+	clkctl0: clkctl@1000 {
+		compatible = "nxp,lpc-syscon";
```

Comment:
This should be something like `compatible = "nxp,rt6xx-syscon";`

Reply:
@galak, why couldn't the compatible attribute reference a common IP block? I believe this is shared between lpc and rt6xx and hence the naming.

Reply:
@dleach02 glancing at the programming models between LPC5xxxx and RT6xx while the blocks are called syscon, the register layouts seem completely different.  Thus we really should have a different `compatible`.